frontend/src/components/auth/login/login.css

```diff
@@ -65,11 +67,12 @@
   box-shadow: inset 2px 2px 5px #888888, inset -2px -2px 5px #ffffff;
 }
 
-.inputLogin:focus{
-  box-shadow: inset 2px 2px 5px #1E488F, inset -2px -2px 5px #1E488F;
+.inputLogin:focus {
```

Comment:
Just make this color same as the linear gradient we are using for input fields click on other pages

Reply:
@Kajol-Kumari  which exact one's color?